### PR TITLE
Unify network api

### PR DESCRIPTION
**Pull request by erkanerol**
Towards https://github.com/giantswarm/giantswarm/issues/26802

### CODE REVIEW

File: helm/cluster-vsphere/templates/kubeadmcontrolplane.yaml [hunk 1]
```diff
@@ -181,10 +181,6 @@ spec:
       {{- end }}
     postKubeadmCommands:
       {{- include "sshPostKubeadmCommands" . | nindent 6 }}
-      {{- if $.Values.network.staticRoutes }}
```
Comment (erkanerol) [inline helm/cluster-vsphere/templates/kubeadmcontrolplane.yaml:184]: this was leftover.

File: helm/cluster-vsphere/values.schema.json [hunk 2]
```diff
@@ -45,23 +59,6 @@
         },
         "controlPlane": {
             "properties": {
-                "endpoint": {
```
Comment (erkanerol) [inline helm/cluster-vsphere/values.schema.json:48]: this was wrong. fixed by moving to the right place.

File: helm/cluster-vsphere/ci/ci-values.yaml [hunk 3]
```diff
@@ -31,16 +31,15 @@ controlPlane:
...
-  loadBalancersCidrBlocks:
-  - "10.10.222.224/30"
+connectivity:
+  network:
+    controlPlaneEndpoint:
+      host: "10.10.222.241"
+      port: 6443
+    loadBalancers:
+      cidrBlocks:
+      - "10.10.222.224/30"
```
Comment (kopiczko) [inline helm/cluster-vsphere/ci/ci-values.yaml:41]: Looks like `podsCidrBlocks` and `servicesCidrBlocks` disappeared?

Comment (erkanerol) [reply helm/cluster-vsphere/ci/ci-values.yaml:41]: Yes. We have default values in the values.yaml so we don't need them here too.

**Review by kopiczko — APPROVED** [review]

**Review by jkremser — APPROVED** [review]
there is that https://github.com/giantswarm/kyverno-policies-connectivity/blob/cc2f344f1607cbfee4549390e6fa574c6f869d1e/helm/kyverno-policies-connectivity/templates/PodController.yaml#L52-L64 but this shouldn't be affected by this change, right? The format of `kubeadmConfig` stays untouched by this.

**Pull request merged**